app/views/admin/budget_investments/_investments.html.erb

```diff
@@ -55,26 +58,9 @@
         <%= investment.valuation_finished? ? t('shared.yes'): t('shared.no') %>
       </td>
       <td class="small">
-        <% if investment.selected? %>
-          <%= link_to toggle_selection_admin_budget_budget_investment_path(@budget,
-                                                                           investment,
-                                                                           filter: params[:filter],
-                                                                           page: params[:page]),
-                      method: :patch,
-                      remote: true,
-                      class: "button small expanded" do %>
-            <%= t("admin.budget_investments.index.selected") %>
-          <% end %>
-        <% elsif investment.feasible? && investment.valuation_finished? %>
-          <%= link_to toggle_selection_admin_budget_budget_investment_path(@budget,
-                                                                           investment,
-                                                                           filter: params[:filter],
-                                                                           page: params[:page]),
-                      method: :patch,
-                      remote: true,
-                      class: "button small hollow expanded" do %>
-            <%= t("admin.budget_investments.index.select") %>
-          <% end %>
+        <% options = investment_selected_link_options(investment) %>
+        <%= link_to toggle_selection_admin_budget_budget_investment_path(@budget,  investment, filter: params[:filter], page: params[:page]), method: :patch, remote: true, class: options[:link_class] do %>
```

Comment:
This line is too long
Try to keep the length of lines to 80 characters or less :ok_hand:
A helper method would be great here or just splitting attributes in separate lines